### ISSUE TITLE
Serial downloading from multiple sources

### ISSUE THREAD

**Issue by fellinir** [komikku-app/komikku#117]
### Describe your suggested feature

I often want to download many chapterals from different sources, and I like to download sequentially, not on parallel, and because of that I am forced to do it one source at the time.
Chapters under the same source can be downloaded one at the time, but if you download multiple sources, those are in parallel.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.10.5](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by cuong-tran** [comment]
As I remember, it currently download 2 entries in parallel, no matter from same source or different source, doesn't it?

**Comment by fellinir** [comment]
@cuong-tran nope. I confirm that currently it downloads more entries in parallel if those are from different sources, but if more entries are from the same source those are only download in serial.
So, i would like to have in the download queue many entries to download from multiple sources and still have them in serial.
